docs/development-guidelines/operational-guidelines.md

```diff
@@ -0,0 +1,99 @@
+
+A technical lead, depending on their seniority and competence, may be the ultimate decision maker and coordinator of all technical efforts, although when templates integrate multiple tools and service, we recommend to scope per technology and have experts on each category assuming this role.
+
+## Case scenarios and specification
+
+Before the team starts coding, make sure to have designed scenarios, use-cases and specification, answering thes questions
+
+- What services, tools and products are we showcasing?
+- What are our technical requirements?
+- What is the use case/functional specification? 
+    * Who will interact with your application?
+    * How does the user interact with the application?
+    * What does the application take as input?
+    * What is the application output?
+- What are the non-functional specifications, apart from the perfomance, security and usability success criteria defined by the standardization effort?
+
+### Creating a backlog
+
+Once you have the use-case and specification in place, create a backlog for all features to be developed, and prioritize them. Assign issues to respective experts, and check the progress on a regular basis.
+Use the backlog to create scopes mapping to [milestones](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/about-milestones), and assign them to the corresponding milestone.
+
+Progress can be shared during standups or async on a Teams chat.
+
+When the issue is completed, close it, referencing the pull request that has addressed it. For example comment and close with `Fixed by #[number of PR]`
+
+## Standardization guidelines
+
+When creating the repository using the automated tool by cloning the [TBD repository], a [Success criteria checklist or definition of done](./definition-of-done.md) issue will be open in your repo. 
+
+After locally working on your application, and before publishing your template to the gallery, automatically or on demand, an automated worflow will run in your repo to validate the standardization success criteria is met.
+
+The system will open related issues when found non-conforming or non-standard blockers in your repository. Once addressed, you can run the validation tool manually, to request reconsideration.
+
+## Version control
+
+Version control ensures code integrity and taceability. Our open-source teams typically collaborate using repositories hosted on Azure-Samples or other organizations.
+
+We advise teams to losely use [GitHub](https://docs.github.com/en/get-started/using-github/github-flow) branching flows and development cycles, always branching out from default branch (default is `main`) and only integrating with a pull-request, with at least 1 reviewer. We strongly recommend you secure your default branch with as a [protected branch](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches)
```

Comment:
Nit: I would drop "losely" here. I think you mean "loosely" but I don't think you need it.

Reply:
Removed!